### PR TITLE
[BugFix][Spark] Fix the bug that VertexWrite does not generate vertex count file

### LINKED ISSUE #109: [Bug][Spark]: The vertex writer does not generate vertex num file when writing a vertex dataFrame
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When writing a vertex DataFrame by VertexWriter, the `vertex_count` file of GAR file format  has not been generated by vertex writer. That's not the expected behavior.

### Expected Behavior

Generated the `vertex_count` file as designed.

### Minimal Reproducible Example

Could use "test vertex writer with only vertex table" case of TestWriter

### Environment

- Operating system:
- GraphAr version:

### Link to GraphAr Logs

_No response_

### Further Information

_No response_

### PR DESCRIPTION
**Pull request by acezen**
## Proposed changes
as title.

## Types of changes

What types of changes does your code introduce to GraphAr?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/alibaba/GraphAr/blob/main/CONTRIBUTING.rst) doc
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments
Fixed #109

### CODE REVIEW

File: spark/src/test/scala/com/alibaba/graphar/TestWriter.scala [hunk 1]
```diff
@@ -61,6 +61,12 @@ class WriterSuite extends AnyFunSuite {
     val chunk_path = new Path(prefix + vertex_info.getPrefix() + "*/*")
     val chunk_files = fs.globStatus(chunk_path)
     assert(chunk_files.length == 20)
+    val vertex_num_path = new Path(prefix + vertex_info.getVerticesNumFilePath())
+    val vertex_num_file = fs.globStatus(vertex_num_path)
+    assert(vertex_num_file.length == 1)
+    val input = fs.open(vertex_num_path)
+    val number = java.lang.Long.reverseBytes(input.readLong())
```
Comment (lixueclaire) [inline spark/src/test/scala/com/alibaba/graphar/TestWriter.scala:68]: we could use FileSystem.readValue here to avoid calling reverseBytes explicitly

Comment (acezen) [reply spark/src/test/scala/com/alibaba/graphar/TestWriter.scala]: Fixed.

File: spark/src/test/scala/com/alibaba/graphar/TestWriter.scala [hunk 2]
```diff
@@ -17,6 +17,7 @@ package com.alibaba.graphar
 
 import com.alibaba.graphar.utils.IndexGenerator
 import com.alibaba.graphar.writer.{VertexWriter, EdgeWriter}
+import com.alibaba.graphar.utils
```
Comment (lixueclaire) [inline spark/src/test/scala/com/alibaba/graphar/TestWriter.scala:20]: modify this line to:
import com.alibaba.graphar.utils.FileSystem
or remove:
import com.alibaba.graphar.utils.IndexGenerator

**Review by lixueclaire — APPROVED** [review]
LGTM, and please fix the minor issue (in the comments) before merging

**Pull request merged**